mccodec/mc/play/world/clientbound_update_advancements.mbt

```diff
@@ -0,0 +1,13 @@
+///|
+/// Update Advancements packet (0x7B)
+/// Updates advancement data
+struct UpdateAdvancementsS2C {
+  reset : Bool // Reset advancements
+  advancement_mapping : Array[(String, String)] // (ID, Data) pairs
```

Comment:
This is a map since advancement ids are unique to its content in Minecraft.

Also, advancement is a complex data structure that requires additional type